test/unit/storage/entities/account.js

```diff
@@ -460,9 +795,69 @@ describe('Account', () => {
 		});
 	});
 
+	describe('_getResults()', () => {
+		const accounts = [
+			new accountFixtures.Account(),
+			new accountFixtures.Account(),
+			new accountFixtures.Account(),
+		];
+
+		before(async () => {
+			await AccountEntity.create(accounts);
```

Comment:
Is it possible that we only use `beforeEach` in every test? Specially when we are changing data and test relies on that data.

Reply:
Done.